product-management/index.md

```diff
@@ -37,15 +49,16 @@ Once the solution and KPIs are agreed upon, update the PRD document.
 
 **Goal: Work with engineering to plan and execute.**
 
-Once the problem, solution, and KPIs are documented, you should share the document with the tech lead and/or engineers and ask him/her to design the solution and help break down the work into small technical tasks with rough estimates. You should create [JIRA](https://www.atlassian.com/software/jira) tickets based on those work items. For planning purposes, we use agile methodology. Our sprints are 2 weeks long and we recommend product manager conduct the following scrum ceremonies:
+Once the problem, solution, and KPIs are documented, you should share the document with the tech lead and/or engineers and ask him/her to design the solution and help break down the work into small technical tasks with rough estimates. You should create [JIRA](https://www.atlassian.com/software/jira) tickets based on those work items. For planning purposes, we use agile methodology. 
+We follow an Agile process with **2-week sprints**, and PMs are expected to lead or participate in the following ceremonies:
 
 -   **Sprint planning (Duration: 60 minutes, When: 2 days before the sprint starts)**: In the sprint planning, you present the work items to the Tech Lead that you would like the engineering team to achieve. This is an opportunity for the Tech Lead to ask questions and for you to get his buyoff on the plan.
 
 -   **Daily standups (Duration: 15 minutes, When: Daily)**: This is a daily touch base call with the product team (engineering, design, QA) driven by Tech Lead. During the meeting, go through the status update and ensure you understand the progress and adapt accordingly to make sure that the overall sprint remains on track. 
 
 - **Feature acceptance (Duration: Ongoing, When: During the sprint)**: During the sprint, you should verify all the PRs and provide "PM Acceptance" for the engineering team to merge and deploy the feature on production. All the reviews and feedback should be submitted within PR on GitHub. 
 
--   **Sprint Exit (Duration: 60 minutes, When: Last day of sprint)**: In the morning of the last day of the sprint, send the engineering team a reminder to ensure that they focus on internal feedback and preparing for the demo on the staging environment. You should also prepare the ["Sprint Review"](https://jalantechnology.sharepoint.com/:p:/s/JTC/Ea0gpWXBwy1LixBUfPo2qNYBrAaK3Ju07bgEkjCbQhZVag?e=1R2YK8) slides summarizing the sprint output.
+-   **Sprint Exit (Duration: 60 minutes, When: Last day of sprint)**: In the morning of the last day of the sprint, send the engineering team a reminder to ensure that they focus on internal feedback and preparing for the demo on the staging environment. You should also prepare a Sprint Review in [Confluence](https://www.loom.com/share/94541e5d80bd4c06815891b46ebf5e31) summarizing the sprint output.
```

Comment:
1. The loom video is a good start but not sufficient. We need to add more walkthrough that shows what a good retro looks and what a good retro does not look like. The video right now is pretty monologue and need to be created from a point of view that its clear "what", "why" and "how" are clear.
2. We also need to keep a copy of the template outside a specific project so its easy for new PM to copy/paste.